kuttl-tests/suites/kuttl-failure-recovery.yaml.template

```diff
@@ -13,5 +13,5 @@ kindContainers:
   - {{ .CASSANDRA_DOCKER_IMAGE }}
   - {{ .RECOVERY_CONTROLLER_DOCKER_IMAGE }}
 {{- end }}
-timeout: 300
+timeout: 360
```

Comment:
300 seconds might be enough, but above timeouts are around 140 seconds per node, leaving only 20 seconds to ensure everything is up and running. 360 seconds seems to be a bit safer.